### DISCUSSION TITLE
How to ignore a specific dependency warning logs on build/dev?

### DISCUSSION THREAD

**Discussion by ubinatus** [vercel/next.js#47968] [Help] [answered]
### Summary

I'm currently using a dependency (context: https://github.com/Urigo/graphql-mesh/issues/2256) that is showing unwanted warning logs that actually doesn't bring potential issues in my application.

**How can we simply ignore specific dependencies warning logs on build/dev compilation?**

<img width="406" alt="image" src="https://user-images.githubusercontent.com/51177379/230124845-f9c7c5eb-e698-4349-b11b-3b71d6be4e10.png">

### Additional information

```js
- Next v13.2.4 (without /app dir)
```

**Comment by icyJoseph** [comment, marked as answer]
If you are using the SWC (that is not .babelrc), then you can use this: https://nextjs.org/docs/advanced-features/compiler#remove-console to silence **_all warnings_**.

To make it specific would be a non-trivial task, unless you actually fix the root issue, or follow the recommendations given to you by the framework.

It seems to be that the issue is that this package loads up dependencies using ternaries or if/else combos, or template strings to resolve the package version. I am not sure if there's a way to tell the bundler to ignore that package.

> @graphql-mesh/config is not compatible [...with...] an environment that uses bundler or something else


This is not specifically Next.js related, though.
